backend/tasks/tasks.py

```diff
@@ -45,7 +57,108 @@ class Lead:
     @property
     def formation_date_str(self):
         return self.formation_date.strftime(DATE_FORMAT)
+    
+def find_CNP_leads() -> int:
+    inserted = 0
+    entities = csv_to_entities('CNPData.csv')
```

Comment:
When I run `docker compose up --build`, I get 

```
File "/app/tasks/tasks.py", line 63, in find_CNP_leads
api_1       |     entities = csv_to_entities('CNPData.csv')
api_1       |   File "/app/tasks/tasks.py", line 124, in csv_to_entities
api_1       |     with open(file_name) as csv_file:
api_1       | FileNotFoundError: [Errno 2] No such file or directory: 'CNPData.csv'
```

I think python is looking for `CNPData.csv` in whatever directory the script is run from and cannot find it. In this kind of scenario, I normally build an absolute path to the data file using the `os` module. So something like

```
path_to_csv = os.path.abspath(os.path.join(
    os.path.dirname(__file__), 
    os.pardir,
    'database',
    'CNPData.csv',
))
```

It's kind of clunky, but usually does the job.

If that doesn't work, then there is a good chance the csv file didn't get copies into the api docker container.